docs/toc.yml

```diff
@@ -28,7 +28,7 @@ items:
     - name: Word add-in
```

Comment:
They should probably be left.  Just because nothing in your content uses the redirect doesn't mean others content points to it or that people no longer have bookmarks to that entry point, 3rd party sites linking to it, etc.

Reply:
I was thinking of removing these that are no longer being used because they were never actually pages and only used in the TOC

Reply:
That makes sense, then. :)